src/UserMenu.tsx

```diff
@@ -119,7 +119,7 @@ export function UserMenu({
           )}
         </Button>
       </TooltipTrigger>
-      {(props) => (
+      {(props: any) => (
```

Comment:
It's probably fine to leave this for a future PR to avoid making this PR too big, but we should consider changing `PopoverMenuTrigger` to take this function as a normal prop rather than as a child, so it's easier to write types for

Reply:
I'm not an expert, but sounds sensible 🙂 I'm definitely not a fan of `any` and weaker types.